### PR TITLE
[4.0] Articles featured postgresql

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
fixed featured sql

### Testing Instructions
install staging 4.x without data
go to  Articles: Featured

### Expected result

No Matching Results 

### Actual result

error
Screenshot from 2019-05-01 12-00-37

### Comments

works as before on mylsql,  pdo mysql

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 7a5adf7eb28e85c6542f807289ea5ed56f1561fa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24768">issues.joomla.org/tracker/joomla-cms/24768</a>.</sub>

**Comment by richard67** [comment]
Drone failure seems to be unrelated to this PR.

File: administrator/components/com_content/Model/FeaturedModel.php [hunk 1]
```diff
@@ -138,8 +138,7 @@ protected function getListQuery()
 					]
 					)
 				)
-				->innerJoin($query->quoteName('#__workflow_stages', 'ws'))
-				->where($query->quoteName('ws.id') . ' = ' . $query->quoteName('wa.stage_id'));
+				->join('INNER', '#__workflow_stages AS ws ON  ' . $query->quoteName('ws.id') . ' = ' . $query->quoteName('wa.stage_id'));
```
Comment (Quy) [inline administrator/components/com_content/Model/FeaturedModel.php]:
```suggestion
				->join('INNER', '#__workflow_stages AS ws ON ' . $query->quoteName('ws.id') . ' = ' . $query->quoteName('wa.stage_id'));
```

**Comment by twister65** [comment]
I have tested this item :white_check_mark: successfully on 7a5adf7eb28e85c6542f807289ea5ed56f1561fa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24768">issues.joomla.org/tracker/joomla-cms/24768</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 9b548e130c0b0a455b17d78a00a4e44f9898dae3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24768">issues.joomla.org/tracker/joomla-cms/24768</a>.</sub>

**Comment by wilsonge** [comment]
@zero-24 @SniperSister rips please

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**